reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/brownfield-byo-bastion.json

```diff
@@ -708,6 +708,9 @@
             },
             { "Key": "openshift-role",
               "Value": "master"
+            },
+            { "Key": "stack-name",
+              "Value": {"Ref": "AWS::StackName"}
```

Comment:
Why not define this at using the tags param on the cloudformation stack? Any tags set at the stack level should filter down to all stack-created objects.